.github/config.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
```

Comment:
Maybe move the location of "If you're reporting a 🐞 bug, please make sure you include steps to reproduce it." a bit.

```
Thanks for opening your first issue here! We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md) and [issue template](https://github.com/videojs/video.js/blob/master/.github/ISSUE_TEMPLATE.md). And if you're reporting a 🐞 bug, please make sure you include steps to reproduce it.
```

Reply:
Only issue I see with this is that if it's in the end, people may drop off before they get there :)